packages/flutter/test/animation/animation_controller_test.dart

```diff
@@ -21,14 +21,17 @@ void main() {
         ..platformDispatcher.onDrawFrame = null;
   });
 
-  test('AnimationController dispatches memory events', () async {
+  test('CurvedAnimation dispatches memory events', () async {
```

Comment:
hey @polina-c I was just passing by and noticed that you edited this test in the wrong file. I think this test should be in `animations_test.dart` file instead. And test from `animations_test.dart` should be here

Reply:
Thank you! Fixed.